### PR TITLE
104389640 chapter 0 master

### PR DESCRIPTION
**Pull request by Serg09**
[#104389640]

### CODE REVIEW

File: intro/Readme.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+### 1) Provide a link in your Readme.md that links to the documentation for the following classes:
```
Comment (jasonnoble) [inline intro/Readme.md:1]: This should be not be part of this PR.  Rebasing will fix?

**Comment by jasonnoble** [comment]
Where's all the chapter_0 changes?

File: intro/Readme.md (hunk 1 above)
Comment (Serg09) [inline intro/Readme.md]: Not sure if I clearly understood the question, but if I just need to remove links I would do updates in Readme.md file and push it to the branch. 
I do not have changes on master and as I understand, I do not need to rebase branch to the master.
???

Comment (Serg09) [inline intro/Readme.md:1]: Should I remove the quastion and leave links only?
